src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -289,7 +289,16 @@ private void addGraalStaticLibsPathToLinkProcess(ProcessBuilder linkBuilder) {
         linkBuilder.command().add(getLinkLibraryPathOption() + getCLibPath());
     }
 
-    private void addJavaStaticLibsPathToLinkProcess(ProcessBuilder linkBuilder) throws IOException {
+    /**
+     * Add the location of the java static libraries to the linker command.
+     * 
+     * <p>By default, the java static libraries will be loaded from {@link FileDeps#getJavaSDKPath()}.
```

Comment:
In order to make windows work, I'm ok with this. But we will have to refactor this.
It is error-prone now that the location of the java static libs is determined by 2 entry points: FileDeps and targets. It seems to me it belongs here, and not in FileDeps. I'll create a follow-up issue to remove it there.